kalite/distributed/hbtemplates/exercise/exercise.handlebars

```diff
@@ -26,7 +26,8 @@
                                     <span class="info-box-header">{{_ "Answer" }}</span>
                                     <ul id="examples" style="display: none"></ul>
                                     <div id="questionpoints"></div>
-                                    <div id="solutionarea" class="fancy-scrollbar"></div>
+                                    <div class="solutionarea-placeholder"></div>
```

Comment:
Khan Academy changed Khan Exercises so it now moves the solution area around, but fails to replace it again after you have finished a question.